ansible/roles/base/tasks/main.yml

```diff
@@ -1,14 +1,12 @@
-
```

Comment:
Hi Rob, i use vim/vi or even notepad lol to edit my files at the mooment.
What editor are you using? Ill download and use the same one to be consistent.